packages/plugin-conversation/src/interceptor.js

```diff
@@ -187,9 +191,10 @@ export default class ConversationInterceptor extends Interceptor {
    * @returns {Promise<Boolean>}
    */
   shouldEncryptRequest(options) {
-    if (!options.method || options.method.toUpperCase() === `GET`) {
+    if (options.shouldEncryptRequest === false || !options.method || options.method.toUpperCase() === `GET`) {
```

Comment:
I feel like we should name this in a negative way (not sure if this is the right word to convey what I mean). That way when `shouldEncryptRequest` is undefined (and therefore falsy) it doesn't seem like we don't want to encrypt. Maybe something like `plaintextRequest` or `skipEncryption` or `omitEncryption`.